tests/test_drs_downloader.py

```diff
@@ -0,0 +1,65 @@
+import os
+import tempfile
+from typing import List
+from drs_downloader.main import download_drs, _get_signed_url, _create_download_url, _extract_tsv_info, _send_request
+
+expectedUrl = 'https://minio-default.aced-idp.org/aced-default/FOOBAR%3Ad62af3ff-756d-4e73-b6c6-e0bbd3bae50d/' \
+    'tests/fixtures/projects/MyFirstProject/DATA/0940858111'
+
+
+def test_download_drs():
+    with tempfile.TemporaryDirectory() as dest:
+        download_drs('tests/gen3-data.tsv', dest)
+        files = next(os.walk(dest))[2]
+        assert len(files) == 10
+
+
+def test_get_signed_url():
+    object_url = 'https://development.aced-idp.org/ga4gh/drs/v1/objects/d62af3ff-756d-4e73-b6c6-e0bbd3bae50d'
```

Comment:
OK for now.  We may need to rethink before release.   (prefer that we don't have to keep a test server up)